src/META-INF/plugin.xml

```diff
@@ -26,7 +26,7 @@
     <p>Alpha pre-release of the 1.0.0 version.</p>
     <p>Doesn't contain all the functionality of the 0.9.x branch but has a completely reworked internals.
       It's faster than 0.9.x, refactoring works to some degree and has native support for gopath packages.</p>
-    
+
```

Comment:
No need to reformat the whole file. As for me, it's better to separate formatting/adding a new functional (just for better git history).